bigtable/docs/snippets.py

```diff
@@ -210,6 +423,69 @@ def test_bigtable_list_app_profiles():
     assert len(app_profiles_list) > 0
 
 
+def test_bigtable_list_tables():
+    # [START bigtable_list_tables]
+    from google.cloud.bigtable import Client
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    tables_list = instance.list_tables()
+    # [END bigtable_list_tables]
+    assert len(tables_list) is not 0
+
+
+def test_bigtable_table_name():
+    import os
+    import json
+    # [START bigtable_table_name]
+    from google.cloud.bigtable import Client
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+
+    table = instance.table(TABLE_ID)
+    table_name = table.name
+    # [END bigtable_table_name]
+
+    cred = os.environ.get('GOOGLE_APPLICATION_CREDENTIALS', 'project_id')
```

Comment:
This might be easier
```python
    import re
    _TABLE_NAME_RE = re.compile(r'^projects/(?P<project>[^/]+)/'
                                r'instances/(?P<instance>[^/]+)/tables/'
                                r'(?P<table_id>[_a-zA-Z0-9][-_.a-zA-Z0-9]*)$')
    assert _TABLE_NAME_RE.match(table_name) in not None
```
If you decide to go with this option, then please remove imports of `os` and `json` above.